stages/org.osbuild.ostree.deploy.container

```diff
@@ -105,9 +107,49 @@ def ostree_container_deploy(tree, inputs, osname, target_imgref, kopts):
             extra_args.append(f'--target-imgref={target_imgref}')
 
             kargs = [f'--karg={v}' for v in kopts]
-
             ostree.cli("container", "image", "deploy",
                        *extra_args, sysroot=tree, *kargs)
+            
+            container_data = subprocess.run(["ostree", "container", "image", "metadata", f"--repo={tree}/ostree/repo", f"{image_source}"],
+					check=True,
+					capture_output=True, 
+					text=True).stdout.rstrip().strip('\"')
+            
+            container_data_config = subprocess.run(["ostree", "container", "image", "metadata", f"--repo={tree}/ostree/repo", f"{image_source}", "--config"],
+					check=True,
+					capture_output=True,
+					text=True).stdout.rstrip().strip('\"')
+            
+            container_data_json = json.loads(container_data)
+            container_data_config_json = json.loads(container_data_config)
+
+            aleph_digest = container_data_json['config']['digest']
+            aleph_ref = f"docker://{target_imgref}@{aleph_digest}"
+            aleph_ostree_version = container_data_config_json['config']['Labels']['version']
+            aleph_ostree_commit = container_data_config_json['config']['Labels']['ostree.commit']
+            aleph_platform = "" # still up for discussion
+            aleph_imgid = "" # still up for discussion
+            aleph_container_image = container_data_config_json['config']['Labels']
+
+            aleph_version_data = {
+                "osbuild-version": osbuild.__version__,
+                "ref": aleph_ref,
+                "build": aleph_ostree_version,
+                "ostree-version": aleph_ostree_version,
```

Comment:
It's not an ostree version, it's just a container image version.  I'd call this field hence just `version`.